lib/internal/crypto/keys.js

```diff
@@ -172,6 +172,9 @@ function isStringOrBuffer(val) {
 }
 
 function parseKeyEncoding(enc, keyType, isPublic, objName) {
+  if (typeof enc !== 'object')
```

Comment:
should this also check for `enc` being `null`?

Reply:
Right, good catch!

Reply:
Perhaps also add a test now that you are at it?